packages/ui/src/working-groups/components/WorkingGroupListItem.tsx

```diff
@@ -24,7 +26,7 @@ export function WorkingGroupListItem({ group }: WorkingGroupProps) {
   return (
     <GroupItem>
       <GroupImageContainer onClick={() => history.push(groupAddress)}>
-        {group.image ? <GroupImage src={group.image} /> : <GroupAvatarPlaceholderImage />}
+        {group.image ? <GroupImage src={group.image} /> : <WorkingGroupImagePlaceholder groupName={group.name} />}
```

Comment:
```suggestion
        {<WorkingGroupImagePlaceholder groupName={group.name} />}
```

We will always take this image. So the `<GroupImage />` can be removed.